src/app/index.ts

```diff
@@ -18,9 +23,18 @@ export const app = async (): Promise<void> => {
     res.status(500).json({ message: err.message });
   });
 
-  app.listen(port, () => {
-    console.log(`App is running on port ${port}`);
-  });
+  const options = { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: true, useCreateIndex: true };
```

Comment:
@hsedaqat1378 please add `ConnectOptions` type to options variable. you can import it from `import { ConnectOptions } from 'mongoose';`